### PR TITLE
Introduce cl_last_version_played, use it to enforcing winter theme

### PR DESCRIPTION
**Pull request by Dune-jr**
Not sure where to put that `DoVersionSpecificActions`, it feels out of place everywhere

### CODE REVIEW

**Comment by Ryozuki** [comment]
Maybe "UpdateConfiguration" is a better name for this?

File: src/game/variables.h [hunk 1]
```diff
@@ -95,7 +95,9 @@ MACRO_CONFIG_INT(ClCameraSpeed, cl_camera_speed, 5, 1, 10, CFGFLAG_CLIENT|CFGFLA
 MACRO_CONFIG_INT(ClShowStartMenuImages, cl_show_start_menu_images, 1, 0, 1, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Show start menu images")
 MACRO_CONFIG_INT(ClSkipStartMenu, cl_skip_start_menu, 0, 0, 1, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Skip the start menu")
 
-MACRO_CONFIG_INT(ClStatboardInfos, cl_statboard_infos, 1259, 1, 2047, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Mask of infos to display on the global statboard")
+MACRO_CONFIG_INT(ClStatboardInfos, cl_statboard_infos, 1259, 1, 2047, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Mask of info to display on the global statboard")
+
+MACRO_CONFIG_INT(ClLastVersionPlayed, cl_last_version_played, 0x073, 0, 0xFFF, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Last version of the game that was played")
```
Comment (oy) [inline src/game/variables.h]: 0x0703 for default value and the max value should be 0xFFFF or we could also drop it and just put a 0 instead, no need to clamp the value.

File: src/game/variables.h [hunk 2]
```diff
@@ -95,7 +95,9 @@ MACRO_CONFIG_INT(ClCameraSpeed, cl_camera_speed, 5, 1, 10, CFGFLAG_CLIENT|CFGFLA
 MACRO_CONFIG_INT(ClShowStartMenuImages, cl_show_start_menu_images, 1, 0, 1, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Show start menu images")
 MACRO_CONFIG_INT(ClSkipStartMenu, cl_skip_start_menu, 0, 0, 1, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Skip the start menu")
 
-MACRO_CONFIG_INT(ClStatboardInfos, cl_statboard_infos, 1259, 1, 2047, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Mask of infos to display on the global statboard")
+MACRO_CONFIG_INT(ClStatboardInfos, cl_statboard_infos, 1259, 1, 2047, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Mask of info to display on the global statboard")
+
+MACRO_CONFIG_INT(ClLastVersionPlayed, cl_last_version_played, 0x0703, 0, 0, CFGFLAG_CLIENT|CFGFLAG_SAVE, "Last version of the game that was played")
```
Comment (fokkonaut) [inline src/game/variables.h]: shouldnt this be CLIENT_VERSION?

Comment (oy) [reply src/game/variables.h]: No, will change `CLIENT_VERSION` when doing a release.

Comment (Dune-jr) [reply src/game/variables.h]: To clear up, we want people who upgrade from 0.7.3 (and older) to have it set to 0.7.3

**Pull request merged**